qiskit/tools/visualization/_state_visualization.py

```diff
@@ -482,23 +563,20 @@ def plot_state(quantum_state, method='city', filename=None, show=False):
             bloch_state = list(
                 map(lambda x: np.real(np.trace(np.dot(x.to_matrix(), rho))),
                     pauli_singles))
-            plot_bloch_vector(bloch_state, "qubit " + str(i), ax=ax)
-        if filename:
-            plt.savefig(filename)
-        elif show:
-            plt.show()
+            plot_bloch_vector(bloch_state, "qubit " + str(i), ax=ax,
+                              figsize=figsize)
     elif method == "wigner":
-        fig = plot_wigner_function(rho, filename=filename, show=show)
+        fig = plot_wigner_function(rho)
```

Comment:
We don't expose this function via `qiskit.tools.visualization.__init__` so we'll drop the functionality when we remove `plot_state()` I'm fine with doing that since this method always felt a bit out of place, and given we need to come to a resolution in #1106 and #1302 I think if we want to make wigner part of the public api we do it as a separate thing.